src/drive/web/modules/drive/Toolbar/components/CreateShortcut.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState, useCallback } from 'react'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import styles from 'drive/styles/toolbar.styl'
+import ShortcutCreationModal from './ShortcutCreationModalConnected'
+
+const CreateShortcutWrapper = () => {
+  const [isModalDisplayed, setIsModalDisplayed] = useState(false)
+  const { t } = useI18n()
+  if (!isModalDisplayed) {
+    return (
+      <a
+        data-test-id="create-a-shortcut"
+        className={styles['fil-action-create-shortcut']}
+        onClick={useCallback(() => {
+          setIsModalDisplayed(true)
+        })}
```

Comment:
I don't mind if it's not in the PR, but I was more thinking of something along the lines of:

```
const toggleModal = useCallback(() => setIsModalDisplayed(! isModalDisplayed))
…
<a onClick={toggleModal}>
…
<ShortcutCreationModal onClose={toggleModal} />
```

(I think hooks are really elegant for this sort of things)